src/viewport/src/Client/ViewportControls.lua

```diff
@@ -25,18 +26,31 @@ function ViewportControls.new(viewport, viewportModel)
 	local self = setmetatable(BaseObject.new(viewport), ViewportControls)
 
 	self._viewportModel = assert(viewportModel, "No rotationYaw")
+	self._enabled = self._maid:Add(ValueObject.new())
```

Comment:
Unfortunately we either need this to be a breaking change and disable by default, OR swap this to true. Either way we should include the type annotation.

```suggestion
	self._enabled = self._maid:Add(ValueObject.new(true, "boolean"))
```